crates/env/src/engine/off_chain/impls.rs

```diff
@@ -277,7 +277,7 @@ impl EnvInstance {
             beneficiary,
             transferred: all,
         };
-        panic!(scale::Encode::encode(&res));
+        std::panic::panic_any(scale::Encode::encode(&res))
```

Comment:
not yet stabilized: ` This is a nightly-only experimental API. (panic_any #78500)`

Also why do we need it?